website/assets/js/pages/customers/new-license.page.js

```diff
@@ -0,0 +1,101 @@
+parasails.registerPage('new-license', {
+  //  ╦╔╗╔╦╔╦╗╦╔═╗╦    ╔═╗╔╦╗╔═╗╔╦╗╔═╗
+  //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
+  //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
+  data: {
+    // Form data
+    formData: { /* … */ },
+
+    // For tracking client-side validation errors in our form.
+    // > Has property set to `true` for each invalid property in `formData`.
+    formErrors: { /* … */ },
+
+    quoteFormRules: {
+      numberOfHosts: {required: true},
+    },
+
+    billingFormRules: {
+      paymentSource: {required: true},
+    },
+
+    // Syncing / loading state
+    syncing: false,
+
+    // Server error state
+    cloudError: '',
+
+    // Success state when form has been submitted
+    cloudSuccess: false,
```

Comment:
Since the other pieces of state below this one are unrelated to the cloud success state, I think they should be separated by an extra newline for clarity (to show they're not related to the comment:

```
// Success state when form has been submitted
```